src/components/radio-button/radio-button.mdx

```diff
@@ -107,6 +109,8 @@ The radio buttons and labels can be disabled using the `disabled` prop on each r
 
 Help text for each input can be added with the `fieldHelp` prop on `RadioButton`. This can be made inline with the `fieldHelpInline` prop.
 
+**Note**: The `fieldHelp` prop is not supported if the `validationRedesignOptIn` flag on the `CarbonProvider` is true.
```

Comment:
**suggestion(non-blocking)**: what do you think about removing this section on the fieldHelp prop, since it is now part of a legacy design?

Reply:
I don't think it hurts to have it further documented somewhere (we've marked it legacy but we've still not had any steer wrt when it actually gets removed etc) but I do agree with the comment elsewhere that we should display the nee designs first and the legacy ones secondary to that.